README.md

```diff
@@ -105,6 +106,20 @@ The prover node requires a configuration file to be passed in at runtime.
 - `server_url` - The URL of the server to connect to for tasks. The provided URL is the dockers reference to the host machines 'localhost'
 - `priv_key` - The private key of the prover node. This is used to sign the tasks and prove the work was done by the prover node.
 
+### HugePages Configuration
+
+It is important to set the hugepages on the host machine to the correct value. This is done by setting the `vm.nr_hugepages` kernel parameter.
+
+For a machine running a single prover node, the value should be set to ~15000. This is done with the following command.
+
+`sysctl -w vm.nr_hugepages=15000`
```

Comment:
So this is only need to be done for the real server? no need anything inside docker?